boards/nordic/nrf5340_audio_dk/nrf5340_audio_dk_nrf5340_cpuapp_common.dtsi

```diff
@@ -120,6 +131,23 @@ arduino_serial: &uart1 {
 	pinctrl-names = "default", "sleep";
 };
 
+arduino_i2c: &i2c1 {
+	compatible = "nordic,nrf-twim";
+	status = "okay";
+	pinctrl-0 = <&i2c1_default>;
+	pinctrl-1 = <&i2c1_sleep>;
+	pinctrl-names = "default", "sleep";
+};
```

Comment:
Instead of this just add `arduino_i2c:` in this line: https://github.com/zephyrproject-rtos/zephyr/blob/708b38d7f5d8dc81a73a90fc05e80a673beea51e/boards/nordic/nrf5340_audio_dk/nrf5340_audio_dk_nrf5340_cpuapp_common.dtsi#L151

Reply:
Good point, will fix.